cmd/erasure-server-pool.go

```diff
@@ -176,7 +163,24 @@ func newErasureServerPools(ctx context.Context, endpointServerPools EndpointServ
 	z.poolMeta.dontSave = true
 
 	// initialize the object layer.
-	setObjectLayer(z)
+	defer setObjectLayer(z)
+
+	// Enable background operations on
+	//
+	// - Disk auto healing
+	// - MRF (most recently failed) healing
+	// - Background expiration routine for lifecycle policies
+	bootstrapTrace("initAutoHeal", func() {
+		initAutoHeal(GlobalContext, z)
```

Comment:
initAutoHeal and probably other are still called from server-main.go here

Reply:
?